domain/src/main/java/life/qbic/domain/usermanagement/policies/EmailFormatPolicy.java

```diff
@@ -36,7 +38,7 @@ public static EmailFormatPolicy create() {
    */
   public PolicyCheckReport validate(String email) {
     if (!honoursRFCSpec(email)) {
-      return new PolicyCheckReport(PolicyStatus.FAILED, "Invalid email address format.");
+      return new PolicyCheckReport(PolicyStatus.FAILED, InvalidEmailMessage);
```

Comment:
Originally i wanted to have a link between the invalid email messages the policy could trigger and the ones recognized by the switch statement in the handler to avoid duplication here.
 But I'm unsure how to do this without violating the clean architecture principles.

Reply:
ok but then you can change it back, right?

Reply:
Sure 👍